sycl/doc/GetStartedGuide.md

```diff
@@ -261,6 +261,17 @@ variable `SYCL_BUILD_PI_HIP_ROCM_DIR` which can be passed using the
 python $DPCPP_HOME/llvm/buildbot/configure.py --hip \
   --cmake-opt=-DSYCL_BUILD_PI_HIP_ROCM_DIR=/usr/local/rocm
 ```
+If further customization is required — for instance when the layout of
+individual directories can not be inferred from `SYCL_BUILD_PI_HIP_ROCM_DIR` —
+it is possible to specify the location of HIP include, HSA include and,
+finally, HIP library directories, using the following CMake variables:
```

Comment:
What does "finally" mean here? Is the order important? If not, I would just remove the word.

Reply:
The final element of the enumeration, but you are right, it does not sound right,  removed.

Reply:
Any chance of getting this approve and merged now this fix in?